bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -136,13 +115,13 @@ Switch  V_DG_OPEN   "Velux DG open"             (gV)    { velux="thing=scene;cha
 Switch  V_DG_SUNNY  "Velux DG sunny"            (gV)    { velux="thing=scene;channel=action#V_DG_Shutters_090" }
```

Comment:
These binding definitions don't match the one given in the "Item Configuration" section above, which says:

````
{ velux="thing=<Mastertype>channel=<Subtype>" }
````

Please review the above definition and correct it.